freqtrade/rpc/telegram.py

```diff
@@ -663,10 +663,10 @@ async def _status_msg(self, update: Update, context: CallbackContext) -> None:
                              ("`({stop_loss_ratio:.2%})`" if r['stop_loss_ratio'] else ""))
                 lines.append("*Stoploss distance:* `{stoploss_current_dist:.8g}` "
                              "`({stoploss_current_dist_ratio:.2%})`")
-                for order in r.get('open_orders', []):
+                if r.get('open_orders'):
                     lines.append(
-                        f"*Open Order:* `{order}`"
-                        + "- `{exit_order_status}`" if r['exit_order_status'] else "")
+                        "*Open Order:* `{open_orders}`"
```

Comment:
We don't need to show details for each order individually?

Reply:
we do - but "open_orders" does NOT contain what you think it does.
it's created in `rpc.py` - which is already combining all orders into one combined string.

Reply:
I see, thank you for the clarification